src/core/ledger/policies/immediate.js

```diff
@@ -18,20 +19,38 @@ export type Immediate = "IMMEDIATE";
 export type ImmediatePolicy = {|
   +policyType: Immediate,
   +budget: G.Grain,
+  +numPeriodsLookback?: number,
```

Comment:
Policies should only have required fields; optionality gets delegated to the config. See https://github.com/sourcecred/sourcecred/pull/2600#issuecomment-755839137

Reply:
Done.